packages/world/src/modules/std-delegations/constants.sol

```diff
@@ -0,0 +1,12 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.0;
+import { ResourceSelector } from "../../ResourceSelector.sol";
+import { ROOT_NAMESPACE } from "../../constants.sol";
+
+bytes16 constant MODULE_NAME = bytes16("stddelegations.m");
+
+// Disposable delegation
+bytes32 constant DISPOSABLE_DELEGATION = bytes32(abi.encodePacked(ROOT_NAMESPACE, bytes16("disposable.d")));
+
+// Timebound delegation
+bytes32 constant TIMEBOUND_DELEGATION = bytes32(abi.encodePacked(ROOT_NAMESPACE, bytes16("timebound.d")));
```

Comment:
should it perhaps have a `MODULE_NAMESPACE`?

Reply:
My thinking was that this module should ideally be installed as a root module (so it can install the systems as root systems), because it reduces the gas cost of all systems that read or change state (around 8k for the calls to `DisposableDelegationSystem`)

Reply:
I don't think I have my head wrapped around the end-to-end, but are these delegation/callfrom checks _always_ routed through the world? Ultimately wondering if it would be useful to inherit the namespace or allow the namespace to be configured, or if that only has negative consequences (only used by world, only more gas)

Reply:
Yes, they are always routed through the World. The main challenge is that the delegation check system needs to know which table to access. We could make this dynamic and store the namespace in which the module is registered in storage and then load it from storage in the system, but that would cost more gas. A cheaper alternative would be to just use two different modules for root and non-root installations (it only needs to be installed once per world, so the namespace for non-root could also be fixed). For now, I'd just go with providing the root version, because I'd expect this standard module to be installed by default - if we find there's a need for a non-root version (to be installed on worlds that don't have the root version installed by default), we can just add another version of the module later

Reply:
root only for now as the default/expected/most common path is totally fine by me